homeassistant/components/reolink/media_source.py

```diff
@@ -94,7 +94,7 @@ async def prepare(self, request: BaseRequest) -> AbstractStreamWriter | None:
 class ReolinkDownloadView(HomeAssistantView):
     """Download VOD View."""
 
-    url = "/api/reolink_download/{config_entry_id}/{channel_str}/{stream_res}/{filename:.*}"
```

Comment:
What is the purpose of the`:.*` ???

Reply:
it means catch remaining, remember some cameras have / in their filenames and if you do not either encode or catch all thats left, you will not get a valid file response.